### ISSUE TITLE
CRAFT: Character Region Awareness for Text Detection

### ISSUE THREAD

**Issue by minha62** [gdscewha-3rd/Study-PaperReview#17]
NAVER의 Clova AI 팀에서 발표한 OCR 논문

`문자` 단위로 영역을 인식해 Text Detection을 하는 모델
모델의 목적: 입력 이미지에 대해 `픽셀`마다 Region score와 Affinity score를 예측
- Region score: 해당 픽셀이 문자의 중심일 확률 (중심에 가까울수록 1, 멀수록 0)
- Affinity score: 해당 픽셀이 인접한 두 문자의 중심일 확률
➡ Affinity score를 기반으로 개별 문자가 하나의 단어로 그룹화될 것인지 결정

<br>

CRAFT 신경망 구조
- Fully Convolutional Network
- VGG-16 기반
- Batch normalization 사용
- U-net처럼 UpConv를 통해 low-level 특징을 집계하여 예측

<br>

한계점
- 각 문자가 분리되어 있지 않고 연결되는 특정 국가의 언어/필기체는 학습이 어렵
- 낮은 FPS

[공식 깃헙](https://github.com/clovaai/CRAFT-pytorch)

**Comment by ssongedev** [comment]
잘 읽었습니다 :)

**Comment by m0oon0** [comment]
아랍어같은건 인식하기 힘들겠네요!! 잘 읽었습니당

**Comment by chaerin314** [comment]
OCR에 관심 있었는데 흥미롭네요! 잘 읽었습니다!
